test/index.js

```diff
@@ -189,7 +189,7 @@ test('Setoid', function() {
   eq(Z.Setoid.test(''), true);
   eq(Z.Setoid.test([]), true);
   eq(Z.Setoid.test({}), true);
-  eq(Z.Setoid.test({'@@type': 'my-package/Quux'}), false);
+  eq(Z.Setoid.test({'@@type': 'my-package/Quux'}), true);
```

Comment:
So this means the object is Setoid as it's a normal object, which is setoid, and the fact that it has `@@type` as it's own prop does not makes it a different object ?